test/cases/edge.sh

```diff
@@ -0,0 +1,350 @@
+#!/usr/bin/bash
+
+#
+# Edge Rest API integration test
+#
+# The test flow in this file is:
+# ------------ Test commit image --------------------------------------------------------
+# 1. Call image builder Rest API. (build an rhel-edge-commit image and upload to aws s3)
+# 2. Download commit image, extract it to /var/www/html, and serve it over httpd
+# 3. Install Edge vm based on commit repo url, and run ansible playbook to check it.
+#
+# ------------ Test installer image ------------------------------------------------------
+# 4. Call image builder Rest API. (build an rhel-edge-install image and upload to aws s3)
+# 4. Download installer image.
+# 5. Install Edge vm with the installer image,and run ansible playbook to check it.
```

Comment:
These 2 scenarios look like they belong to the osbuild-composer repository. 

I am not sure why do we want to add them inside the image-builder repository.

Reply:
Yes, we have almost same test in osbuild-composer, but this is to make sure that the Edge images built by image-builder can work well.
Actually we did find issues, the edge commit tar works well, but the installer ISO image has problem, bug is reported at: https://github.com/osbuild/image-builder/issues/206

Reply:
Hmm, I want to discuss this more on the QE meeting. It starts to feel like we're doing some redundant testing and I'd like to know what's the root cause of the issue you reported and why we didn't catch this with the existing tests in osbuild-composer. From the looks of it we should have.

Reply:
Ok, we can discuss in the QE meeting.
And just heads up, I ever asked this question to Achilleas, and seems he is also trying to find out why osbuild-composer can build installer ISO image successfully but it hang in the image-builder.